pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1253,6 +1299,7 @@ func appNetworkDoActivateUnderlayNetwork(
 	setNetworkACLRules(ctx, appID, ulStatus.Name, ruleList)
 
 	if appIPAddr != "" {
+		checkIPRange(netInstStatus.DhcpRange.Start, net.ParseIP(appIPAddr), netInstStatus.DhcpRange.End)
```

Comment:
Are any of the warnings reachable, i.e. should the code generating/parsing application IP address be made more robust? Or is this just to be informed if we introduce any bugs in this area in the future (like during the refactoring)?

Reply:
It is reachable, saw it when I tested it manually.

Reply:
Could you give an example of an input that triggers this, I will try to improve this area in zedrouter during refactoring.

Reply:
It was only triggered because I had a wrong check in `checkIPRange`.